tests/scripts/make_bad_replica.sh

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env bash
+
```

Comment:
The script is missing `set -eo pipefail`, as a result the script doesn't always fail when one of the commands fails, e.g.
`../tests/scripts/make_bad_replica.sh: line 123: docker: command not found`

Reply:
We'll need to think some more about how to create bad replicas these because `docker` isn't always available in the test environment. E.g. when using VScode, the tests are running in another Docker container that doesn't have Docker installed